pandas/tests/test_frame.py

```diff
@@ -13542,6 +13542,14 @@ def test_round(self):
             with self.assertRaises(TypeError):
                 df.round(nan_round_Series)
 
+        # named columns
+        df2 = df.copy()
+        df2.columns.name = "cols"
+        tm.assert_equal(df2.round().columns.name, "cols")
```

Comment:
instead of doing this, just add the names onto the ``expected_rounded`` guy (and the original df). the ``assert_frame_equal`` automatically checks for names (since they are both ``None`` it didn't make a difference before though)